cou/utils/openstack.py

```diff
@@ -464,3 +409,48 @@ def is_charm_supported(charm: str) -> bool:
     :rtype: bool
     """
     return bool(OpenStackCodenameLookup.lookup(charm)) or charm in SUBORDINATES
+
+
+def _generate_track_mapping() -> dict[TrackKeys, str]:
+    """Generate the track mapping for the auxiliary charms.
+
+    This mapping should be updated periodically by adding new lines in the file
+    openstack_to_track_mapping.csv
+
+    See the following url for more details:
+    https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html
+
+    :return: Dictionary containing the tracks by charm name, series and OpenStack release.
+    :rtype: dict[namedtuple, str]
+    """
+    track_mapping = {}
+    with open(
+        Path(__file__).parent / "openstack_to_track_mapping.csv",
+        encoding=encodings.utf_8.getregentry().name,
```

Comment:
`encodings.utf_8.getregentry().name` is just... 'utf-8'. Why not use the string directly?

Reply:
I ask the same question @esunar and he provide sufficient answer.

Reply:
Since all "UTF-8", "UTF8", "ut", "utf8", "utf-8", "utf-8", "utf_8" and a bunch of other strings can be used as 'utf-8'. Unfortunately python does not provide a simple constant.

Reply:
weird...